src/wazuh_db/wdb_parser.c

```diff
@@ -4056,6 +4065,80 @@ int wdb_parse_ciscat(wdb_t * wdb, char * input, char * output) {
     }
 }
 
+int wdb_parse_rootcheck(wdb_t * wdb, char * input, char * output) {
+    char * curr;
+    char * next;
+    int result = 0;
+    next = wstr_chr(input, ' ');
```

Comment:
If `input` is NULL, this will crash.

Reply:
This is already verified before calling `wdb_parse_rootcheck`. I have now added _`_attribute__((nonnull))` just to be sure